scrapy_autounit/utils.py

```diff
@@ -146,6 +166,7 @@ def write_test(fixture_path):
 
 class AutoUnit(unittest.TestCase):
     def test_{spider_name}_{callback_name}_{fixture_name}(self):
+        self.maxDiff = None
```

Comment:
Maybe this could be a setting

Reply:
That sounds reasonable, but python itself doesn't do it maybe for some reason?  You could also wait until someone raises an issue with it, then you'd have a concrete use case.

Reply:
So do you mean we shouldn't make it a setting yet or that we should remove the maxDiff set to None?

Reply:
Ah, I was saying making it a setting or keeping the `= None` both sound fine.